learn-ocaml.opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "learn-ocaml"
-version: "0.10"
+version: "0.11"
```

Comment:
Why did you bump the version number?

Reply:
Because I introduced a change in the API, older clients won't be compatible with the new server (since I require a hash of the secret).